### PR TITLE
Improved dynamic disk sizing content

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
# What changed, and why it matters

Improved the content for Dynamic disk sizing.

### CODE REVIEW

File: docs/platform/concepts/dynamic-disk-sizing.rst [hunk 1]
```diff
@@ -1,7 +1,9 @@
-Dynamic Disk Sizing
+Dynamic disk sizing
 ====================
 
-When running a service, you may soon fill up the storage available and notice that the other resources are not limited. Upgrading your plan is an option but then you are paying for additional compute resources (or RAM) that your current usage does not require. If your need is to exclusively increase the storage, with Dynamic Disk Sizing (DDS) you can achieve it for the following services: 
+Dynamic disk sizing (DDS) allows you to add or remove additional storage to your services from within the `Aiven Console <https://console.aiven.io/>`_. You can add additional storage during service creation or on-demand without interruption to meet the growing business needs.
```
Comment (ftisiot) [inline docs/platform/concepts/dynamic-disk-sizing.rst:4]:
```suggestion
Dynamic Disk Sizing (DDS) allows you to add or remove additional storage to your services from within the `Aiven Console <https://console.aiven.io/>`_. You can add additional storage during service creation or on-demand without interruption to meet the growing business needs. 
```

File: docs/platform/concepts/dynamic-disk-sizing.rst [hunk 2]
```diff
@@ -1,7 +1,9 @@
-Dynamic Disk Sizing
+Dynamic disk sizing
```
Comment (ftisiot) [inline docs/platform/concepts/dynamic-disk-sizing.rst:1]: If the feature is DDS should we always uppercase when we talk about it?
```suggestion
Dynamic Disk Sizing
```

File: docs/platform/concepts/dynamic-disk-sizing.rst [hunk 3]
```diff
@@ -10,24 +12,53 @@ When running a service, you may soon fill up the storage available and notice th
...
-With Dynamic Disk Sizing, you can add up to 2x the original storage amount to any plan, allowing you 3x the initial storage amount. 
+.. note::
 
-For example, if you are deploying a ``business-4`` Aiven for PostgreSQL service, the default storage is 80GB, and you can add up to 160GB of additional storage, to give a total of 240GB.
+    Dynamic Disk Sizing is not supported on custom plans.
 
-The price you see is the cost of the storage, as well as any backups associated with it.
+Why dynamic disk sizing?
+------------------------
+When creating a service on Aiven, you have a fixed pre-defined storage allocation based on your plan selection. However, when running your Service, there is a possibility that you may soon realize the need for scaling up the storage alone.
```
Comment (ftisiot) [inline docs/platform/concepts/dynamic-disk-sizing.rst:21]:
```suggestion
When creating a service on Aiven, you have a fixed pre-defined storage allocation based on your plan selection. However, when running your service, there is a possibility that you may soon realize the need for scaling up the storage alone.
```

File: docs/platform/concepts/dynamic-disk-sizing.rst [hunk 4]
```diff
@@ -10,24 +12,53 @@ When running a service, you may soon fill up the storage available and notice th
...
+- It is cost-effective, as you pay only for the additional storage, not the compute resources that would have been part of an upgraded service plan. 
 
 How does Dynamic Disk Sizing work?
 ----------------------------------
+You can :doc:`add additional disk storage <../howto/add-storage-space>` to your service plan when you create a new service on the `Aiven Console <https://console.aiven.io/>`_ or on-demand when you need more storage while running your service. Adding additional storage will not affect or interrupt the operations of your service. 
+
+When you add additional storage to your service, the Aiven platform provisions additional disk storage and dynamically adds it to your running instances. The total amount of additional storage you can add to your service is based on your service plan and the cloud provider.
+
+In a clustered service (Cassandra or Kafka), the additional storage added will be equally divided between the nodes. In a shared service, each node receives the total shared capacity of the additional storage added. All the additional storage added will remain usable.
```
Comment (ftisiot) [inline docs/platform/concepts/dynamic-disk-sizing.rst:39]:
```suggestion
In a clustered service (Apache Cassandra or Apache Kafka), the additional storage added will be equally divided between the nodes. In a shared service, each node receives the total shared capacity of the additional storage added. All the additional storage added will remain usable.
```

File: docs/platform/concepts/dynamic-disk-sizing.rst [hunk 5]
```diff
@@ -10,24 +12,53 @@ When running a service, you may soon fill up the storage available and notice th
...
+- Each time additional storage is added to the running service, an additional disk is added to the corresponding nodes. Since the cloud providers may limit the maximum number of disks attached to a node, this limits the number of times you can add additional storage. If this limit is reached, you will see a prompt to perform a maintenance update for performance optimization. You can add additional storage two times until the maximum storage limit is reached.  Therefore, we advise you to plan to add a sufficient amount of disk capacity. 
+- If there is an ongoing Maintenance update, you will not be able to add additional storage until the maintenance is completed. 
+
 
-When increasing storage, the Aiven platform provisions additional network attached storage and dynamically adds it to your running instances. Your services are not affected or interrupted and you can continue to use your service as normal. 
+Next steps
+----------
+For infromation on how to add additonal disk storage to your service from the `Aiven Console <https://console.aiven.io/>`_, see :doc:`Add additional storage <../howto/add-storage-space>`.
```
Comment (ftisiot) [inline docs/platform/concepts/dynamic-disk-sizing.rst:58]: There are a couple of warnings here regarding spelling

**Review by ftisiot — CHANGES_REQUESTED** [review]
Hi @harshini-rangaswamy good piece, minor comments from my side

File: docs/products/postgresql/howto/monitoring-query-performance [hunk 6]
```diff
@@ -0,0 +1,34 @@
+Monitoring Query Performance with Datadog
```
Comment (ftisiot) [inline docs/products/postgresql/howto/monitoring-query-performance:1]: Hi @harshini-rangaswamy, why is this file here?

File: docs/platform/howto/create_new_service.rst [hunk 7]
```diff
@@ -28,12 +28,14 @@ To create a new service:
 
       This defines the number of servers and what kind of memory, CPU, and disk resources are allocated to your service.
 
-      For **Aiven for Apache Kafka®**, **Aiven for PostgreSQL®**, and **Aiven for MySQL** services, you can also select **Add disk storage** to configure additional storage on top of the amount defined in your selected service plan.
+   d. Add additional disk space
+      
+      Configure additional disk storage for your service by using the slider and selecting the desired additional disk capacity. You can also see the cost for the additional storage on the right side and in the Service summary section on the console.
```
Comment (ftisiot) [inline docs/platform/howto/create_new_service.rst:33]: Should we add a link to the concept about dynamic-disk-sizing?

File: docs/platform/howto/add-storage-space.rst [hunk 8]
```diff
@@ -1,25 +1,55 @@
...
-Compared to service plan upgrades, setting additional disk storage is mainly intended for cases where you do not need additional computing power.
+Add additional storage during new service creation
+--------------------------------------------------
+You can add additional storage when creating a new service. For information on how to add additional storage during new service creation, see :doc:`Create a new service <../howto/create_new_service>`. 
 
-.. note::
-   This feature is not available for all service plans. For example, **Aiven for Apache Kafka®** *Startup* and **Aiven for PostgreSQL®** *Hobbyist* service plans do not support adding storage space.
+Add additional storage to running service
+-----------------------------------------
+You can add additional storage to your running service from the Aiven web console without interrupting the service.
```
Comment (ftisiot) [inline docs/platform/howto/add-storage-space.rst:13]: Add a link to the related concept?

**Review by ftisiot — CHANGES_REQUESTED** [review]
Hi @harshini-rangaswamy, very minor changes needed. 
But I spot a file (monitoring query performance in Datadog) that shouldn't be here?

File: docs/products/postgresql/howto/monitoring-query-performance (hunk 6 above)
Comment (harshini-rangaswamy) [reply docs/products/postgresql/howto/monitoring-query-performance:1]: This shouldn't be in here! Thanks for spotting this one.

**Review by ftisiot — APPROVED** [review]
LGTM! Good work
